src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -2293,15 +2293,6 @@ module Queries = struct
         |> Runtime_config.to_yojson |> Yojson.Safe.to_basic )
 
   let fork_config =
-    let rec map_results ~f = function
```

Comment:
`map_results ~f xs` should be equivalent to `List.map ~f xs |> Result.all`

Reply:
Yes, but it's less efficient, as it iterates over the list twice, doesn't it?